docs/source/concepts/ledger-model/ledger-integrity.rst

```diff
@@ -226,7 +229,103 @@ becomes apparent.
 `A`'s commit is not conformant to the contract model, as the model does
 not contain the top-level action she is trying to commit.
 
-.. _`da-model-authorization`:
+.. _da-model-key-consistency:
+
+Key Consistency
++++++++++++++++
+
+Contract keys introduce two validity constraints for ledgers: :ref:`key uniqueness <def-key-uniqueness>` and :ref:`key assertion validity <def-key-assertion-validity>`.
+To capture these notions, the contract model must specify for every contract in the system whether the contract has a key and, if so, the key.
+Every contract can have at most one key.
+Additionally, the model must provide the function `maintainers` that takes a key and returns the non-empty set of maintainers of the key.
```

Comment:
Is it worth explaining what a maintainer is in terms of what the ledger model expects from them?

Reply:
Moved to the authorization section along with an explanation.